cmd/ctr/snapshot.go

```diff
@@ -195,9 +196,14 @@ var removeSnapshotCommand = cli.Command{
 
 var prepareSnapshotCommand = cli.Command{
 	Name:      "prepare",
-	Usage:     "prepare gets mount commands for digest",
-	ArgsUsage: "[flags] <digest> <target>",
-	Flags:     []cli.Flag{},
+	Usage:     "prepare a snapshot from a tar file by digest",
```

Comment:
This usage string does not make sense now since the tar is not involved.Just `prepare a snapshot from a committed snapshot` is enough